File: Lab 5/t2.c
#include <stdio.h>

int main (void)
{
  int height = 0, weight = 0;

  scanf ("%d %d", &height, &weight);

  if ((height - 105) > (weight + 20)) {
    puts ("You are too slim");
  } else {
    if (((weight - 10) < (height - 105)) && ((height - 105) <= (weight + 20))) {
      puts ("You are fit");
    } else {
      puts ("You are too fat");
    }
  }

  return 0;
}

File: Lab 5/t2.go
package main

import "fmt"

func main() {
  var height, weight int = 0, 0

  fmt.Scanf("%d %d", &height, &weight)

  if (height - 105) > (weight + 20) {
    fmt.Println("You are too slim")
  } else if ((weight - 10) < (height - 105)) && ((height - 105) <= (weight + 20)) {
    fmt.Println("You are fit")
  } else {
    fmt.Println("You are too fat")
  }
}
